docker/deployed/staging/api/config.json

```diff
@@ -8,7 +8,8 @@
   },
   "Gateway": {
     "ExternalURIPrefix": "https://staging.tableland.network",
-    "MetadataRendererURI": ""
+    "MetadataRendererURI": "",
+    "AnimationRendererURI": ""
```

Comment:
We don't have an app to render `staging` data, this means the value is set to an empty string. This implies the response of the `GET /chain/{chainID}/tables/{id}` call won't include the `animation_url` property. See [omitempty](https://github.com/tablelandnetwork/go-tableland/pull/332/files#diff-6ca5d243b7947601125912e4889dbdd233c1c22aefcedee8b18a10e83e972e1bR45) in the the struct definition.